plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/MavenFacetImpl.java

```diff
@@ -237,9 +237,7 @@ private void putArtifact(final MavenPath path, final Payload payload, final Stor
       componentAttributes.set(P_GROUP_ID, coordinates.getGroupId());
       componentAttributes.set(P_ARTIFACT_ID, coordinates.getArtifactId());
       componentAttributes.set(P_VERSION, coordinates.getVersion());
-      if (coordinates.isSnapshot()) {
-        componentAttributes.set(P_BASE_VERSION, coordinates.getBaseVersion());
-      }
+      componentAttributes.set(P_BASE_VERSION, coordinates.getBaseVersion());
```

Comment:
Always store baseVersion. This seems like redundant step, but allows for uniform SQL and code (no need for branching if baseVersion null or not).